wear/src/main/java/io/homeassistant/companion/android/home/MainViewModel.kt

```diff
@@ -108,6 +108,8 @@ class MainViewModel @Inject constructor(
         private set
     var templateTileRefreshInterval = mutableStateOf(0)
         private set
+    var isFavoritesOnly = mutableStateOf(false)
```

Comment:
All settings are using `=`, but wouldn't it be nice to use `by` instead so you don't need to use `.value` everywhere?

Reply:
Sure, let me change this new one I added here and then lets update them all in a follow up PR since that will touch a few different places :)